nixos/tests/tandoor-recipes.nix

```diff
@@ -21,7 +21,11 @@ import ./make-test-python.nix ({ lib, ... }: {
       ensureUsers = [
         {
           name = "tandoor_recipes";
-          ensurePermissions."DATABASE tandoor_recipes" = "ALL PRIVILEGES";
+          ensurePermissions = {
+            "DATABASE tandoor_recipes" = "ALL PRIVILEGES";
+            # Needed for running migrations
+            "SCHEMA public" = "ALL PRIVILEGES";
```

Comment:
I am unclear on what the security implications are for this permission: would comprimising `tandoor-recipes` allow an attacker to drop other services' tables?

Reply:
This doesn't sound like a good "solution" to document in a test, putting this on draft until I can change the test to use SQLite.

This can be reverted once work on #266270 is done.